luigi/local_target.py

```diff
@@ -128,19 +128,16 @@ def rename_dont_move(self, path, dest):
         self.move(path, dest, raise_if_exists=True)
 
 
-class LocalTarget(FileSystemTarget):
+class LocalTarget(FileSystemTarget, LazyFormatMixin):
```

Comment:
Same here?

Reply:
err, I think maybe the context for this was on another comment? not sure what you were referrring to :/